config/exampleconfiguration.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using the configuration library is simple. Import the configure_settings function and call it. The only requirement for succesful collection is to have an existing project.yml file in the current directory of the notebooks. \n",
+    "\n",
+    "### This example project has one, so we will call configure_settings from there. \n",
+    "\n",
+    "### Further usage require us to load the settings. Instead of having the user have to import other libraries, we expose a second function from notebook_config called get_settings() which will return an instance of ProjectConfiguration. \n",
+    "\n",
+    "### To complete this example, we will obtain an instance and print out settings values. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from notebook_config import configure_settings, get_settings"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Now that the functions are imported, lets bring up the UI to configure the settings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "configure_settings()"
```

Comment:
Didn't see this, but I actually already added something. We'll need the file to exist, regardless, but if the sub id has already been provided it won't pop up. This way, for automated testing, we will bypass the pop up because we can write the file out first. .